common/persistence/visibility/store/elasticsearch/visibility_store.go

```diff
@@ -350,29 +350,29 @@ func (s *VisibilityStore) AddBulkRequestAndWait(
 
 	if err != nil {
 		if errors.Is(err, context.DeadlineExceeded) {
-			return &persistence.TimeoutError{Msg: fmt.Sprintf("visibility task %s timed out waiting for ACK after %v", visibilityTaskKey, s.processorAckTimeout())}
+			return &persistence.TimeoutError{Msg: fmt.Sprintf("visibility task timed out waiting for ACK after %v", s.processorAckTimeout())}
```

Comment:
This is main change of this PR. Having task key in error message makes it impossible to group log lines in loki. All required information (TaskId, TaskType, WorkflowId, and RunId) is added as tags by logger.